modules/bg_controller/lamba.tf

```diff
@@ -0,0 +1,30 @@
+# Writes received taskTokekn to SSM. 
```

Comment:
Same comment as for file appmesh-sf-task-token.py

Reply:
if it appears in master how come it appears as a change ?